game.cpp

```diff
@@ -1,2 +1,84 @@
 #include <bits/stdc++.h>
-using namespace std;
\ No newline at end of file
+#include "main-character.cpp"
+#include "monster-generator.cpp"
+#include "battle.cpp"
+#include "store.cpp"
+#include "ranking.cpp"
+
+using namespace std;
+
+const string INTRODUCTION = "BEM VINDO AO XIAQ\n";
+const string CONGRATULATIONS_MESSAGE = "Parabéns!! Você ganhou mais uma batalha..\n";
+const string START_OPTIONS_MESSAGE = "Digite 1 caso queira batalhar, 2 se quiser ver as instruções ou 3 para visualizar o ranking.\n";
```

Comment:
As default, its probably better to show the options as something like:
"Menu Principal:\n[1] Iniciar Jogo\n[2] Ver Instruções\n[3] Ver Recordes"
We've already set this pattern in other files (store, battle)